automated-comments/java/hamming/should_use_stream_filter_and_count.md

```diff
@@ -0,0 +1 @@
+Stream-based computations should make use of Stream's `filter` and `count` methods.
```

Comment:
I'm wondering about "should." As a student, I might be wondering if I was expected to change anything or not. Is the student required to make a change? If so, maybe rephrase to something like the comment below.